docs/static/docker.asciidoc

```diff
@@ -0,0 +1,108 @@
+[[docker]]
+== Running Logstash on Docker
+Docker images for Logstash are available from the Elastic Docker registry.
+
+Obtaining Logstash for Docker is as simple as issuing a +docker pull+ command
+against the Elastic Docker registry.
+
+ifeval::["{release-state}"=="unreleased"]
+
+However, version {logstash_version} of Logstash has not yet been released, so no Docker
+image is currently available for this version.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+The Docker image for Logstash v{logstash_version} can be retrieved with the following
```

Comment:
Remove the "v" before v{logstash_version} (we don't use the convention of prefacing version numbers with a "v".